frontend/pages/create-project.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useState, useEffect } from 'react';
+import { PhotoIcon } from '@heroicons/react/24/solid'
+
+function CreateProject() {
+  const [projectName, setProjectName] = useState('');
+  const [aboutProject, setAboutProject] = useState('');
+  const [selectedFile, setSelectedFile] = useState<File | null>(null);
+  const [material, setMaterial] = useState<string[]>([]);
+  const [materialsList, setMaterialList] = useState<string[]>([]);
+  const [newMaterial, setNewMaterial] = useState('');
+  const [listOfParticipant, setLists] = useState<string[]>([]);
+  const [newList, setNewList] = useState('');
+  const [error, setError] = useState<string>('');
+  const maxCharacterLimit = 1000;
```

Comment:
Do not use all os these `useState`, take a look a packages such as `formik` made to handle forms